README.md

```diff
@@ -1,3 +1,31 @@
+* *Status:* Production
```

Comment:
I think these lines should go under the mainline header instead of first. This looks bad for an external README. Maybe we should talk to Lisa about specific guidelines for external repos.

Also, I prefer the button versions of this info since it is not so overwhelming. Feel free to look at this PR for examples:

https://github.com/seomoz/simhash-cpp/pull/12